src/main/java/picard/vcf/GatherVcfs.java

```diff
@@ -45,9 +48,24 @@
 
     @Argument(shortName = StandardOptionDefinitions.OUTPUT_SHORT_NAME, doc = "Output VCF file.")
     public File OUTPUT;
+    
+    @Argument(doc = "Comment(s) to include in the merged output file's header.", optional = true, shortName = "CO")
+    public List<String>  COMMENT = new ArrayList<>();
+    
+    @Argument(doc = "Re-order the INPUT files according to order of their first variant.", optional = true, shortName = "S")
```

Comment:
This documentation is unclear...I needed to read the code to understand what you mean.

Explain that the program will assume that the variants in the files come from non overlapping regions, but that while each file is itself sorted, the order that the files are provided in is not to be trusted. If TRUE, instead of sorting all the variants, the program will concatenate the variants from the files using the order of the first variant in each file as the order of the files. 

Also, "SORT" is not a good name, since it's is used elsewhere to mean that the outputs will be sorted....I suggest you use REORDER_INPUT_BY_FIRST_VARIANT